sound/soc/sof/ipc.c

```diff
@@ -424,15 +423,14 @@ static void ipc_trace_message(struct snd_sof_dev *sdev, u32 msg_id)
 static void ipc_period_elapsed(struct snd_sof_dev *sdev, u32 msg_id)
 {
 	struct sof_ipc_stream_posn posn;
+	struct snd_sof_pcm_stream *stream;
 	struct snd_sof_pcm *spcm;
-	u32 posn_offset;
 	int direction;
 
 	/* check if we have stream box */
 	if (sdev->stream_box.size == 0) {
```

Comment:
isn't this also completely Intel-specific and dependent on mailboxes?

Reply:
Sure, they have to be removed too. This patch-set doesn't remove *all* Intel-specific code or objects from SOF, it's the next step in that direction.